src/compiler/emitter.ts

```diff
@@ -3055,17 +3055,34 @@ var __param = (this && this.__param) || function (paramIndex, decorator) {
             }
 
             function emitVariableStatement(node: VariableStatement) {
+                let variableDeclarations = <VariableDeclaration[]>node.declarationList.declarations;
                 let startIsEmitted = true;
-                if (!(node.flags & NodeFlags.Export)) {
-                    startIsEmitted = tryEmitStartOfVariableDeclarationList(node.declarationList);
+                if (node.flags & NodeFlags.Export) {
+                    if (!isES6ExportedDeclaration(node)) {
+                        variableDeclarations = filter(variableDeclarations, node => !!node.initializer);

```

Comment:
>Write a new function outside of this one named emitIfInitializerIsPresent which takes a VariableDeclaration. When you're emitting an exported declaration, but you're not in ES6 mode, then instead of emitCommaList, we're going to use emitList directly (you can omit the null-check in emitCommaList in this case).

I'm not sure I understand? Don't you need some kind of for loop then if it accepts `VariableDeclaration`?